concent_api/core/payments/validation.py

```diff
@@ -0,0 +1,19 @@
+from core.exceptions import PaymentTimestampError
+from core.payments.backends.sci_backend import TransactionType
+from core.payments.service import get_list_of_payments
+
+
+def validate_that_last_closure_time_is_older_than_oldest_payment(
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    search_payments_since_ts: int,
+) -> None:
+    forced_payment_event_list = get_list_of_payments(  # pylint: disable=no-value-for-parameter
+        requestor_eth_address=requestor_eth_address,
+        provider_eth_address=provider_eth_address,
+        payment_ts=search_payments_since_ts,
+        transaction_type=TransactionType.FORCE,
+    )
+    if forced_payment_event_list != [] and search_payments_since_ts < max([event.closure_time
+                                                                           for event in forced_payment_event_list]):
```

Comment:
The pull request is now closed but I just spotted it so I'll post anyway.

This wrapping is against our coding style. In my opinion this way of wrapping things is not very readable and indenting code like this is not very convenient. I'd do it as:

``` python
if forced_payment_event_list != [] and search_payments_since_ts < max(
    [event.closure_time for event in forced_payment_event_list]
):
```

or, better:
``` python
closure_times = [event.closure_time for event in forced_payment_event_list]

if len(closure_times) > 0 and search_payments_since_ts < max(closure_times):
```

You also generally should not compare with `[]`. What if it was not a `list` but some type inherited from it? Or a `tuple`? A set? `len(forced_payment_event_list) > 0` is way more robust and just as readable.